lib/src/components/NxTextLink/__tests__/NxTextLink.test.tsx

```diff
@@ -5,114 +5,106 @@
   });
 
   it('adds the noreferrer rel if the noReferrer prop is true', function() {
-    expect(getShallow()).toHaveProp('rel', '');
-    expect(getShallow({ rel: 'foo' })).toHaveProp('rel', 'foo');
+    expect(renderEl()).toHaveAttribute('rel', '');
+    expect(renderEl({ rel: 'foo' })).toHaveAttribute('rel', 'foo');
 
-    expect(getShallow({ noReferrer: true })).toHaveProp('rel', 'noreferrer');
+    expect(renderEl({ noReferrer: true })).toHaveAttribute('rel', 'noreferrer');
 
-    expect(getShallow({ noReferrer: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
-    expect(getShallow({ noReferrer: true, rel: 'foo' }).prop('rel')).toContain('foo');
+    expect(renderEl({ noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
+    expect(renderEl({ noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
   });
 
   it('adds the noreferrer rel if the external prop is true and noReferrer is not false', function() {
-    expect(getShallow({ external: true })).toHaveProp('rel', 'noreferrer');
+    expect(renderEl({ external: true })).toHaveAttribute('rel', 'noreferrer');
 
-    expect(getShallow({ external: true, rel: 'foo' }).prop('rel')).toContain('foo');
-    expect(getShallow({ external: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
+    expect(renderEl({ external: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
+    expect(renderEl({ external: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
 
-    expect(getShallow({ external: true, noReferrer: true })).toHaveProp('rel', 'noreferrer');
-    expect(getShallow({ external: true, noReferrer: false })).toHaveProp('rel', '');
+    expect(renderEl({ external: true, noReferrer: true })).toHaveAttribute('rel', 'noreferrer');
+    expect(renderEl({ external: true, noReferrer: false })).toHaveAttribute('rel', '');
 
-    expect(getShallow({ external: true, noReferrer: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
-    expect(getShallow({ external: true, noReferrer: true, rel: 'foo' }).prop('rel')).toContain('foo');
+    expect(renderEl({ external: true, noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
+    expect(renderEl({ external: true, noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
   });
 
   it('sets aria-disabled on the text link when the disabled prop is set', function() {
-    expect(getShallow()).toHaveProp('aria-disabled', false);
-    expect(getShallow({ disabled: true })).toHaveProp('aria-disabled', true);
+    expect(renderEl()).toHaveAttribute('aria-disabled', 'false');
+    expect(renderEl({ disabled: true })).toHaveAttribute('aria-disabled', 'true');
   });
 
   it('sets link role on the text link when the disabled prop is set', function() {
-    expect(getShallow()).not.toHaveProp('role', 'link');
-    expect(getShallow({ disabled: true })).toHaveProp('role', 'link');
+    expect(renderEl()).not.toHaveAttribute('role', 'link');
```

Comment:
Not sure if this is relevant or not, but the test will pass with any kind of role attached to NxTextLink as long as it's not 'role'. If there isn't supposed to be a role attached to the link unless it's disabled, perhaps it could be `expect(renderEl()).not.toHaveAttribute('role')`. As I said though, not sure if testing whether ANY role is applied prior to disabling NxTextLink is part of testing

Reply:
> Not sure if this is relevant or not, but the test will pass with any kind of role attached to NxTextLink as long as it's not 'role'. If there isn't supposed to be a role attached to the link unless it's disabled, perhaps it could be `expect(renderEl()).not.toHaveAttribute('role')`. As I said though, not sure if testing whether ANY role is applied prior to disabling NxTextLink is part of testing

Updated here: 7910971123c15e7e2be37ae5dd3a497e0ffa19ec